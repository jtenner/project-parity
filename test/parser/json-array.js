const { OptionType } = require("../../lib");

module.exports = {
  argv: ["--option-a", "test/files/d.json,test/files/e.json"],
  config: {
    optionA: {
      type: OptionType.JSONArray,
    },
  },
};
