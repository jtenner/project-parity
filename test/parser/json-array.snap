{
  args: [ 'test\\parser\\json-array.config.js' ],
  cliProvided: Set(1) { 'optionA' },
  configProvided: Set(0) {},
  errors: [],
  options: {
    optionA: [
      [ 1, 2, 3, 4, {} ],
      { f: { g: { h: {} } } }
    ]
  },
  unknown: [ '--config' ]
}
